### ISSUE TITLE
Unrecognized option "refresh_jwt" under "security.firewalls.api_token_refresh"

### ISSUE THREAD

**Issue by maxlech974** [markitosgv/JWTRefreshTokenBundle#260]
Hi,

During installation, i've got this message:  

```
In ArrayNode.php line 314:
                                                                                                                                                                                                       
  Unrecognized option "refresh_jwt" under "security.firewalls.api_token_refresh". Available options are "access_denied_handler", "access_denied_url", "anonymous", "context", "custom_authenticators"  
  , "entry_point", "form_login", "form_login_ldap", "guard", "host", "http_basic", "http_basic_ldap", "json_login", "json_login_ldap", "jwt", "lazy", "lexik_jwt", "login_link", "login_throttling",   
  "logout", "methods", "pattern", "provider", "remember_me", "remote_user", "request_matcher", "required_badges", "security", "stateless", "switch_user", "user_checker", "x509".  
```

Unrecognized option "refresh_jwt" under "security.firewalls.api_token_refresh",

when trying to do bin/console make:migration . I have to remove this line to continues steps, but i think it will not working on any more!

`refresh_jwt: ~`

Any idea about this trouble?

**Comment by mbabker** [comment]
What Symfony version?  The `refresh_JWT` is only made available in Symfony 5.3+.

**Comment by pmikolajek** [comment]
I've encountered the same problem. In my case, it was because I was using stable version of the library (`0.12`), while looking at the docs from `master` branch. `refresh_jwt` is not yet available in the stable version.

**Comment by maxlech974** [comment]
> What Symfony version? The `refresh_JWT` is only made available in Symfony 5.3+.

I am on 5.3




> I've encountered the same problem. In my case, it was because I was using stable version of the library (`0.12`), while looking at the docs from `master` branch. `refresh_jwt` is not yet available in the stable version.

Thank for your answer, finally i manage to refresh manually my token! ;)
